receiver/kafkametricsreceiver/documentation.md

```diff
@@ -9,6 +9,15 @@ These are the metrics available for this scraper.
 | Name | Description | Unit | Type | Attributes |
 | ---- | ----------- | ---- | ---- | ---------- |
 | **kafka.brokers** | Number of brokers in the cluster. | {brokers} | Gauge(Int) | <ul> </ul> |
+| **kafka.brokers.consumer_fetch_rate_avg** | Average consumer fetch Rate | {fetches}/s | Gauge(Double) | <ul> <li>broker</li> </ul> |
```

Comment:
do we really need `_avg` suffix? I would recommend removing it from all the metrics